hugofs/glob/glob.go

```diff
@@ -117,14 +107,50 @@ func (g globDecorator) Match(s string) bool {
 	return g.g.Match(s)
 }
 
+type globDecoratorDouble struct {
+	primary   glob.Glob
+	secondary glob.Glob
+}
```

Comment:
I think renaming the above to `lowerCase` and `originalCase` would make it easier to understand.